discord/app_commands/translator.py

```diff
@@ -190,6 +219,11 @@ async def translate(self, string: locale_str, locale: Locale, context: Translati
             The locale being requested for translation.
         context: :class:`TranslationContext`
             The translation context where the string originated from.
+
+            .. note::
+                For better type checking ergonomics, the ``TranslationContextTypes``
+                type can be used instead to aid with type narrowing. It is functionally
+                equivalent to :class:`TranslationContext`.
```

Comment:
This could just be regular text instead of an admonition.
```suggestion
            For better type checking ergonomics, the ``TranslationContextTypes``
            type can be used instead to aid with type narrowing. It is functionally
            equivalent to :class:`TranslationContext`.
```